doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,36 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html)
+iceoryx keeps things simple and offers only basic Quality of Service (QoS) settings.The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are similar to the respective DDS setting:
+
+| DDS                       | iceoryx                             | Explanation                                                                                                                    |
+|---------------------------|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| History depth of writer   | `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                            |
+| History depth of reader   | `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored before they are passed to the user                                       |
+| -                         | `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, must be equal or smaller than `historyCapacity` |
+
+The history depth of DDS depend on the durability setting. Late-joining readers in DDS
+only get values with `Non-Volatile` durability, e.g. `Transient Local`.
```

Comment:
I agree that we should not explain (or even mention) the DDS concepts here. At most there existence (but even that is a stretch). At the level it can be explained here (not much detail) it does not really help and is not relevant to an iceoryx user I would argue.

If at one point we want a more detailed comparison we should devote a whole document to it. While probably not needed, the reader can just skip it if he is not interested.